tests/src/case0054.c

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2009-2020, Christian Ferrari <tiian@users.sourceforge.net>
+ * All rights reserved.
+ *
+ * This file is part of LIXA.
+ *
+ * LIXA is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as published
+ * by the Free Software Foundation.
+ *
+ * LIXA is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with LIXA.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include <config.h>
+
+
+
+#ifdef HAVE_STDIO_H
+# include <stdio.h>
+#endif
+#ifdef HAVE_UNISTD_H
+# include <unistd.h>
+#endif
+#ifdef HAVE_ASSERT_H
+# include <assert.h>
+#endif
+
+
+
+#include <tx.h>
+#include <lixa_tx.h>
+#include <liblixamonkey.h>
+
+
+
+/* this is a special case test: it is a basic test for statically registered
+   resource managers with a code-specified profile supplied as an argument */
+
+
+
+int main(int argc, char *argv[])
+{
+    char *pgm = argv[0];
+    char *profile = NULL;
+    int rc;
+    TXINFO info;
+
+    if (argc > 0) {
+        profile = argv[1];
+    }
+    
```

Comment:
Please add some more stressful testing like for example setting it multiple times.

Reply:
Extra test cases added here and to TX_3_2.at.

Reply:
It sounds very good. I'm not able to test it due to function g_steal_pointer that breaks the build chain. It keep it open as a reminder to test it later.

Reply:
The test passes even on my build system.

Reply:
I am not quite sure what you mean by this, but the case0054 program is used for 3 test cases, and with no parameters it will use LIXA_PROFILE so will work if this has an appropriate value.

Reply:
It was positive feedback!